solr/solr-ref-guide/modules/upgrade-notes/pages/major-changes-in-solr-7.adoc

```diff
@@ -230,7 +230,7 @@ Mentions of deprecations are likely superseded by removal in Solr 7, as noted in
 
 Note again that this is not a complete list of all changes that may impact your installation, so a thorough review of CHANGES.txt is highly recommended if upgrading from any version earlier than 6.6.
 
-* The Solr modules `map-reduce`, `morphlines-core` and `morphlines-cell` have been removed.
+* The Solr Modules `map-reduce`, `morphlines-core` and `morphlines-cell` have been removed.
```

Comment:
Hmm, historic release notes should probably not be changed. This was probably an oversight in 9.0 while bulk search/replacing `s/contrib/module/`. See https://solr.apache.org/guide/8_6/major-changes-in-solr-7.html#major-changes-in-earlier-6-x-versions for original text.
```suggestion
* The Solr contribs `map-reduce`, `morphlines-core` and `morphlines-cell` have been removed.
```

Reply:
thanks!